01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,11 @@
+1.upto(20) { |i|
+  if i % 3 == 0 && i % 5 == 0
+    puts 'FizzBuzz'
+  elsif i % 3 == 0
+    puts 'Fizz'
+  elsif i % 5 == 0
+    puts 'Buzz'
+  else
+    puts i
+  end
+}
```

Comment:
ブロックが複数行に渡るときは`do ~ end`で書くことが多いような気がしますね。今の書き方でも間違いではないですが、参考までに。

```suggestion
1.upto(20) do |i|
  if i % 3 == 0 && i % 5 == 0
    puts 'FizzBuzz'
  elsif i % 3 == 0
    puts 'Fizz'
  elsif i % 5 == 0
    puts 'Buzz'
  else
    puts i
  end
end
```